pkgs/development/web/valum/default.nix

```diff
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation rec {
   pname = "valum";
-  version = "0.3.15";
+  version = "0.3.16";
 
   src = fetchFromGitHub {
     owner = "valum-framework";
     repo = "valum";
     rev = "v${version}";
-    sha256 = "1wk23aq5lxsqns58s4g9jrwx6wrk7k9hq9rg8jcs42rxn2pckaxw";
+    sha256 = "15lnk91gykm60rv31x3r1swp2bhzl3gwp12mf39smzi4bmf7h38f";
   };
 
```

Comment:
I would add mutliple outputs to clean up what gets place in the outpaths
```suggestion
  outputs = [ "out" "dev" "doc" ];
```
original:
```
$ tree ./results/valum/
./results/valum/
├── lib
│   ├── libvalum-0.3.so
│   ├── libvsgi-0.3.so
│   ├── pkgconfig
│   │   ├── valum-0.3.pc
│   │   └── vsgi-0.3.pc
│   └── vsgi-0.3
│       └── servers
│           ├── libvsgi-cgi.so
│           ├── libvsgi-fastcgi.so
│           ├── libvsgi-http.so
│           ├── libvsgi-mock.so
│           └── libvsgi-scgi.so
└── share
    └── doc
        └── valum-0.3
            └── api
...
10 directories, 462 files
```

after:
```
$ tree ./result
./result
└── lib
    ├── libvalum-0.3.so
    ├── libvsgi-0.3.so
    └── vsgi-0.3
        └── servers
            ├── libvsgi-cgi.so
            ├── libvsgi-fastcgi.so
            ├── libvsgi-http.so
            ├── libvsgi-mock.so
            └── libvsgi-scgi.so

3 directories, 7 files
```
saves a bit on the closure size:
```
[nix-shell:/home/jon/.cache/nix-review/pr-68623]$ nix path-info -Sh ./results/valum
/nix/store/av7kwpqz06p1ha5a50xb2j6gawx7hxin-valum-0.3.16          50.2M
[nix-shell:/home/jon/.cache/nix-review/pr-68623]$ nix path-info -Sh ./result
/nix/store/sm5fhchwwwxljv7z8fsfng4sn848098a-valum-0.3.16          48.3M
```

Reply:
@jonringer does anything in nixpkgs use this that would then need adjusting? I'm mostly just focusing on ZHF at the moment and don't want to get too distracted improving the quality of broken packages. A good number of the broken packages have been broken for some time, implying they are older and didn't adhere today what is today known as best practice... so I can see spending so much time improving packages that I lose out on time to actually breakfix more packages. The benefit of merely fixing a broken package is that I don't need to weigh compatibility or functionality considerations too much. If the package compiles (and doesn't introduce security concerns) generally we're in a better state than "it doesn't build and I don't know why".

I'd also like to specifically state that I always appreciate @jonringer and any input from you :smile: You are proposing a good change here, but I'm trying to blow through as many ZHF packages as I can during my lunch hour and after putting the kids to bed.

If anyone uses `valum` I hope they will step up and maintain this package as the original author is long gone, so this is currently an orphan package. Alternatively, if you tell me you've tested this and are reasonably sure it won't cause problems I'm happy to hit `Apply suggestion` as I always trust the good word of @jonringer :+1:

Reply:
I know the maintainers want to get everything cleaned up for the release, that makes sense. Don't mind me. :). Otherwise it looked good to me :)

Reply:
as far as i can tell, no other package consumes this library, so I don't know a good way to test. But it builds on linux

Reply:
if i got pinged I would have helped out with ZHF, guess it's unfortunate that all my packages build :(

Reply:
If you're interested and have the capacity feel free to grab a broken package and get it building: https://github.com/NixOS/nixpkgs/issues/68361.

Reply:
one step ahead of you buddy :)